tests/job_applications/tests.py

```diff
@@ -198,6 +199,38 @@ def test_accept_follow_up_group(self):
         assert membership.creator == user
 
 
+@pytest.mark.parametrize(
+    "factory, override_sender, validation_should_pass",
+    [
+        (JobApplicationSentByJobSeekerFactory, None, True),
+        (JobApplicationSentByJobSeekerFactory, JobSeekerFactory, False),
```

Comment:
```suggestion
        (lambda: JobApplicationSentByJobSeekerFactory(sender=JobSeekerFactory()), False),  # inconsistent job_seeker/sender
```
et on se débarasse du `override_sender`